### PR TITLE
Mining Textures & More: Revival

### PR DESCRIPTION
**Pull request by Sun-Soaked**
## About The Pull Request
- A fixed version of the PR https://github.com/shiptest-ss13/Shiptest/pull/429, containing all it's content wholesale, albeit with some minor pathing adjustments to hopefully fix previous merge conflicts.
- Also includes the previous mining suit reskin as a new mining suit subtype.
- Gives holoparasites very slightly increased damage absorption and the ability to fly in zero g.

I'll work on some implementation stuff when the research PR is merged and I can touch maps again.

## Why It's Good For The Game
Fixes a number of bugs I accidentally caused with previous mining PRs
Adds new and purty mining textures, as well as a bit of setting fleshing and some new loot.
Jackhammers now function as demolition tools as well as mining ones, mimicking how they're used in real life.

## Changelog
:cl:
add: The Syndicate Cleaver, a syndiepilled crusher option. You've never seen anything like it before. Ever. Don't make them sue.
add: legacy mining equipment, slightly more powerful versions of typical exploration gear that also cause mild slowdown.
add: The Heavy Mining Suit, for heavy-duty resource extraction jobs.
add: new multi-functional equipment for the syndicate Brigador cyborg, as well as a new sprite.
add: varied references to EXOCON, a new-age frontier operations corp.
add: jackhammer wall and window smashing functionality.
add: an exocon-themed secure crate.
add: some showcases, to be used in a mapping project.
tweak: set up a few new ss13 archetype corpses, usable in mapping
tweak: added some backpack loot to legion miner drops, and an Old Miner corpse drop.
balance: the legion staff now fires significantly weaker skulls.
balance: boosts the power of several holoparasites, gives holoparasites spacewalking.
fix: replaced ice demon crusher loot drops to prevent runtimes
fix: fixed a continuous clown simplemob-related runtime
imageadd: added a number of new mining-related sprites and recolors
imageadd: ported resprites of the singularity hammer and mjolnir
imageadd: added marg.
imageadd: an ominous wired-up brigador. Where could it possibly be used? find out on the next episode of dragon shiptest z
spellcheck: fixed some minor typos and spacing issues here and there
add: marg.
/:cl:

### CODE REVIEW

**Comment by Sun-Soaked** [comment]
> albeit with some minor pathing adjustments to hopefully fix previous merge conflicts.

aaaa

**Comment by Apogee-dev** [comment]
Considering the introduction of a different mining hardsuit in #455, perhaps it might make more sense to flavor the original mining hardsuit as Nanotrasen-issue, since the intent is to replace them on most non-nanotrasen ships anyway? Or to otherwise figure out how we're going to place these three hardsuits relative to each other, I suppose.

**Comment by Sun-Soaked** [comment]
I worked about with this for a while, and I feel like the Heavy Mining suit is ultimately intended to be a premium mining suit you buy ig through cargo/it's appends, as opposed to a starter suit. 
I'll implement a cargo buy for it, and buff it slightly.

**Comment by Apogee-dev** [comment]
> I worked about with this for a while, and I feel like the Heavy Mining suit is ultimately intended to be a premium mining suit you buy ig through cargo/it's appends, as opposed to a starter suit. I'll implement a cargo buy for it, and buff it slightly.

Yeah, I got that impression about the heavy suit through its description and stats. I was mostly concerned about the standard mining hardsuit being reworked to be more of a generic mass-produced one rather than a Nanotrasen special, since that's the niche the one in the indie suit bundle is intended to fill.

**Comment by Sun-Soaked** [comment]
if I'm being honest, the basic suit could use a rework to look more like an NT design, as well. Currently it feels like something people just kind of wear, like the t-shirt of the frontier- it's also kind of just a really noisy sprite without much actually going on.
Ideally it would look more like the Old Exploration Gear in this pr, in terms of flavour and color.

**Comment by Sun-Soaked** [comment]
image
(some examples of that)

**Comment by Apogee-dev** [comment]
> if I'm being honest, the basic suit could use a rework to look more like an NT design, as well. Currently it feels like something people just kind of wear, like the t-shirt of the frontier- it's also kind of just a really noisy sprite without much actually going on. Ideally it would look more like the Old Exploration Gear in this pr, in terms of flavour and color.

I think it does effectively accomplish the original idea behind it- which is to say, it's a Dead Space rigsuit. They're kind of noisy by nature, and redesigning it would lose the instant recognizability of the "classic" miner hardsuit that most other codebases have. At the moment I do think the nanotrasen hardsuits, by and large, are pretty recognizable and have some common features, like the triple vision slits, that make them recognizably Nanotrasen. The only real exception to that that isn't normally a unique item is the medic hardsuit, which frankly does need a redesign to better fit with the other NT suits.

**Comment by triplezeta** [comment]
ill resprite the mining hardsuit as faithfully as i can at a later date

**Comment by triplezeta** [comment]
along with every other hardsuit most of them kind of smell

**Comment by Apogee-dev** [comment]
> along with every other hardsuit most of them kind of smell

They could do with a little more clarity. The engineering hardsuit is the best of them, i really dislike the sec suit and prefer the design used on vg (which isn't just native server bias, the palette's a lot better and it's easier to read the suit's shape)

**Comment by Sun-Soaked** [comment]
image
what did they mean by this

**Comment by triplezeta** [comment]
image
it means this

**Comment by Sun-Soaked** [comment]
please father,  I don't want to manually remove and re-import all my icons again

**Comment by Sun-Soaked** [comment]
HOW

File: code/game/objects/structures/window.dm [hunk 1]
```diff
@@ -190,6 +190,16 @@
 				to_chat(user, "<span class='notice'>You pry the window into the frame.</span>")
 			return
 
+	if(istype(I, /obj/item/pickaxe/drill/jackhammer))
+		to_chat(user, "<span class='notice'>You begin to smash though [src]...</span>")
+		if(do_after(user, 75, target = src))
+			if(!istype(src, /obj/structure/window))
+				return TRUE
```
Comment (Fikou) [inline code/game/objects/structures/window.dm:197]: jesse

Comment (Sun-Soaked) [reply code/game/objects/structures/window.dm:197]: This is behavior inherited from other jackhammer smash procs I ported from tg, you'll see they have a similar define. I think it prevents a bug related to triggering the effect on other objects after clicking a window?
Not entirely sure, but I made sure to preserve it.

**Comment by Sun-Soaked** [comment]
wooo yeah woooo yeah woooo yeah

**Comment by Apogee-dev** [comment]
A small request, but could we get a legion corpse type with the asteroid miner jumpsuit + indie mining hardsuit? Wanted to add that to my own PR before it got merged but i honestly expected this one to be merged first.

**Comment by Sun-Soaked** [comment]
:videogames:

**Pull request merged**